src/core/MOM.F90

```diff
@@ -191,7 +200,8 @@ module MOM
                                      !! set by calling the function useRegridding() from the
                                      !! MOM_regridding module.
   logical :: do_dynamics             !< If false, does not call step_MOM_dyn_*. This is an
-                                     !! undocumented run-time flag that is fragile.
+  logical :: do_online               !< If false, does not call step_MOM_dyn_*. This is an
```

Comment:
Compiles but doesn't run uniformly. I'm getting uninitialize behavior I think because this logical is never set. You read DO_ONLINE in the solo driver but I don't see where you pass it to MOM or read and set it at the level of MOM.F90.